packages/nds-stacked-nav/README.md

```diff
@@ -1,11 +1,169 @@
+
+The text that appears as the component's heading. If the `label` is omitted, the heading section of the component will not render at all.
+
+###### elementType
+
+- Type: `React.ElementType`
+- Default: `p`
+
+The tag that you would like to use to wrap the label for the heading. The default is a paragraph tag which implies no heirarchial structure.
+
+###### link
+
+- Type: `object`
+- Default: `null`
+
+An object to describe an optional link for the heading of the stacked nav component.
+
+###### link.elementType
+
+- Type: `React.ElementType`
+- Default: `a`
+
+The tag that you would like to use for the heading link. By default it's an HTMLAnchorElement and will result in something like `<a href="/destination">Stacked Nav Heading</a>` but you can pass a custom router link tag such as `Link` to result in `<Link to="/destination">Stacked Nav Heading</Link>`.
+
+###### link.destination
+
+- Type: `string`
+- Default: `null`
+
+A string to represent the destination if the heading is a link.
+
+###### link.isCurrent
+
+- Type: `boolean`
+- Default: `false`
+
+A boolean value to determine whether the link should have style and ARIA attributes to show that it is currently active.
+
+##### `<StackedNavLink />`
+
+###### children
+
+- Type: `string`
+
+Can be used instead of the label property.
+
+> The label property takes precedence over a child string
+
+```jsx
+<StackedNavLink>About Page</StackedNavLink>
+```
+
+###### label
+
+- Type: `string`
+
+The value that appears as the link's text, takes precedence over a supplied child.
+
+```jsx
+<StackedNavLink label="This will appear!">This won't appear</StackedNavLink>
+```
+
+###### elementType
+
+- Type: `React.ElementType`
+- Default: `a`
+
+The tag that you would like to use for the link. By default it's an HTMLAnchorElement and will result in something like `<a href="/destination">Destination</a>` but you can pass a custom router link tag such as `Link` to result in `<Link to="/destination">Destination</Link>`.
+
+###### destination
+
+- Type: `string`
+- Default: `null`
+
+A string to represent the destination if the element is a link.
+
+###### isCurrent
+
+- Type: `boolean`
+- Default: `false`
+
+A boolean value to determine whether the link should have style and ARIA attributes to show that it is currently active.
+
+###### hint
+
+- Type: `string`
+- Default: `null`
+
+A string can be passed to the `hint` property to display some additional text underneath the main label (but still inside the link element).
+
+### SCSS
+
+If you're not using [React](#react), then import the SCSS directly into your application by:
+
+```scss
+@import "~@nice-digital/nds-stacked-nav/scss/stacked-nav";
 ```
-const ndsStackedNav = require('@nice-digital/nds-stacked-nav');
 
-// TODO: DEMONSTRATE API
+### HTML
+
+If you're not using [React](#react), then include the [SCSS as above](#scss) and use the HTML:
+
+```html
+<nav class="stacked-nav" aria-label="Stacked Nav Heading">
+	<h1 class="stacked-nav__root">
```

Comment:
I think example should default to using a `p` tag because I don't think we want a heading here 'normally'